--- CDVWKWebViewEngine.m.orig	2019-04-06 05:39:41.273110588 +0200
+++ CDVWKWebViewEngine.m	2019-04-07 13:45:38.469345720 +0200
@@ -127,6 +127,13 @@
            selector:@selector(onAppWillEnterForeground:)
                name:UIApplicationWillEnterForegroundNotification object:nil];
 
+    // check for keyboardWillHide event to workaround https://github.com/WebKit/webkit/commit/0ff63c48b7456d6b72b0e751a600d9e3d14dbdeb regression
+    NSLog(@"%@", @"Observing keyboardWillHide event to workaround https://github.com/WebKit/webkit/commit/0ff63c48b7456d6b72b0e751a600d9e3d14dbdeb regression");
+    [[NSNotificationCenter defaultCenter]
+        addObserver:self
+           selector:@selector(keyboardWillHide:)
+               name:UIKeyboardWillHideNotification object:nil];
+
     NSLog(@"Using WKWebView");
 
     [self addURLObserver];
@@ -163,6 +170,18 @@
     }
 }
 
+- (void) keyboardWillHide:(NSNotification*)notification {
+    if (@available(iOS 12.0, *)) {
+        WKWebView *webview = (WKWebView*)_engineWebView;
+        for(UIView* v in webview.subviews) {
+            if ([v isKindOfClass:NSClassFromString(@"WKScrollView")]) {
+                UIScrollView *scrollView = (UIScrollView*)v;
+                [scrollView setContentOffset:CGPointMake(0, 0)];
+            }
+        }
+    }
+}
+
 - (BOOL)shouldReloadWebView
 {
     WKWebView* wkWebView = (WKWebView*)_engineWebView;
